main.go

```diff
@@ -47,7 +48,11 @@ func main() {
 		config.Cmd = strings.Split(os.Getenv("CMD"), " ")
 	}
 	create := docker.CreateContainerOptions{Name: containerName, Config: &config}
-	hostConfig := docker.HostConfig{PublishAllPorts: true}
+
+	portBindings := map[docker.Port][]docker.PortBinding{
+        docker.Port(publishPort + "/tcp"): []docker.PortBinding{docker.PortBinding{HostPort: publishPort }}}
```

Comment:
We should use the same syntax as the docker commandline interface.
`format: ip:hostPort:containerPort | ip::containerPort | hostPort:containerPort | containerPort`